src/main/scala/cromwell/binding/types/WdlArrayType.scala

```diff
@@ -0,0 +1,20 @@
+package cromwell.binding.types
+
+import cromwell.binding.values.{WdlInteger, WdlArray}
+import spray.json.JsArray
+
```

Comment:
Is this part of optional parameters or separate? (the latter is fine IMO, just for my own understanding)